### ISSUE TITLE
Anti-Ro/SS-A antibody positivity

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6583]
**Preferred term label:** Anti-Ro/SS-A antibodies

**Synonyms**

**Definition (free text, please give PubMed ID)**
Anti-Ro/SS-A antibodies alone are found in 50–70% of Sjögren's syndrome and 30% of SLE with cutaneous involvement. PMID 1522616

**Parent term (use hpo.jax.org/app)**
Anti-extractable nuclear antibody positivity (when created)

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Systemic Lupus Erythematosus OMIM:152700
Sjogren Syndrome OMIM:270150

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by eswietlik** [comment]
Note that there is a term Anti-La/SSA antibody positivity HP:0032235 under Autoimmune antibody positivity. I would remove this term or move under Anti-ANA > Anti-ENA Ab and amend. There are two types of these antibodies although in practice they are usually detected together: Anti-Ro/SS-A and anti-La/SS-B, the current term is a weird combination of both.

**Comment by pnrobinson** [comment]
done, thanks. I have amended the Anti-La/SSA term as suggested
